### DISCUSSION TITLE
Add functionality to save/label a collection of drives

### DISCUSSION THREAD

**Discussion by pntzio** [teslamate-org/teslamate#1706] [Ideas]
It would be useful to be able to save a collection of drives and label it.

For example a weekend trip would consist of multiple drives, and saving them as a collection with the label "My cool weekend trip" would allow me to quickly find it without remembering the dates and times (this can be done in "Trips", if you know the datetime interval).

For me it would be a useful feature as I then can quickly look back on trips and compare efficiency/prices/times etc.

**Comment by Skaning** [comment]
+1 on that suggestion. Very useful to be able to combine multiple trips in to one.

**Comment by toneus** [comment]
This may do what you're asking. Set an Absolute Time Range in the Time picker, and then save that as a new Dashboard. Any custom Time Range is also remembered and will be presented in the Time picker as a Recently Used Time Range.

![image](https://user-images.githubusercontent.com/12435503/121528674-b360b100-c9c9-11eb-8cc7-75544e36014d.png)
